MdeModulePkg/Universal/DisplayEngineDxe/InputHandler.c

```diff
@@ -747,6 +747,10 @@ GetNumericInput (
 
     case '+':
     case '-':
+    if (ManualInput && ((NumericOp->Flags & EFI_IFR_DISPLAY) == EFI_IFR_DISPLAY_UINT_DEC)) {
+       UpdateStatusBar(INPUT_ERROR, TRUE);
+       break;
+     }

```

Comment:
```suggestion
      if (ManualInput && ((NumericOp->Flags & EFI_IFR_DISPLAY) == EFI_IFR_DISPLAY_UINT_DEC)) {
        UpdateStatusBar(INPUT_ERROR, TRUE);
        break;
      }
```

Reply:
right, it always gives me trouble if the indents are not 4*n spaces, will fix